content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst

```diff
@@ -0,0 +1,184 @@
+======================
+Requests for quotation
+======================
+
+.. _inventory/purchase/manage_deals/request_for_quotation:
```

Comment:
Small change to this custom anchor, for convention's sake–

- Update the app/module to Purchase, not Inventory
- Can abbreviate to "RFQ" to make the anchor shorter


```suggestion
.. _purchase/manage_deals/rfq:
```